doc/lin/index.mld

```diff
@@ -5,14 +5,14 @@
 - {!module-Lin} is a base module for specifying sequential consistency tests.
 - {!module-Lin_domain} exposes a functor that allows to test a library in
   parallel (with domains).
+- {!module-Lin_thread} exposes a functor that allows to test a library in
+  concurrency (with threads).
 - {!module-Lin_effect} exposes a functor that allows to test a library in
   concurrency (with effects).
```

Comment:
There's still two occurrences of *in concurrency* which reads a bit strange to me